deisctl/units/deis-swarm-node.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=deis-swarm-node
+
+[Service]
+EnvironmentFile=/etc/environment
+TimeoutStartSec=20m
+ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/swarm` && docker history $IMAGE >/dev/null 2>&1 || docker pull $IMAGE"
```

Comment:
Shouldn't this be `/run/deis/bin/get_image /deis/swarm-node` or are they the same Docker image?